src_assets/common/assets/web/configs/tabs/Advanced.vue

```diff
@@ -72,7 +72,7 @@ const config = ref(props.config)
           </template>
           <template #windows>
             <option value="ddx">Desktop Duplication API</option>
-            <option value="wgc">Windows.Graphics.Capture {{ $t('_common.beta') }}</option>
+            <option value="wgc">Windows Graphics Capture API</option>
```

Comment:
```suggestion
            <option value="wgc">Windows.Graphics.Capture</option>
```

This is the correct name to use.